Example/Sources/DesignSystem/TooltipViewController.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  TooltipViewController.swift
+//  Example
+//
+//  Copyright (c) 2021 Virtusize KK
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import UIKit
+import Foundation
+import VirtusizeUIKit
+
+class TooltipViewController: UIViewController {
+	private var tooltip1: VirtusizeUITooltip?
+	private var tooltipParamsArray: [VirtusizeUITooltipParams] = []
+	private var tooltipParamsArrayIndex = 0
+
+	private var tooltip2: VirtusizeUITooltip?
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		view.backgroundColor = .white
+		title = "Tooltip Example"
+
+		let tooltip1Button = VirtusizeUIButton()
+		tooltip1Button.setTitle("Show Tooltip", for: .normal)
+		tooltip1Button.translatesAutoresizingMaskIntoConstraints = false
+		view.addSubview(tooltip1Button)
+		tooltip1Button.addTarget(self, action: #selector(showTooltip1), for: .touchUpInside)
+
+		let tooltip2Button = VirtusizeUIRoundButton()
+		tooltip2Button.translatesAutoresizingMaskIntoConstraints = false
+		view.addSubview(tooltip2Button)
+		tooltip2Button.addTarget(self, action: #selector(showTooltip2), for: .touchUpInside)
+
```

Comment:
@akueisara Could we find better descriptive variable name than 1 and 2?

Reply:
@zr0z Yes, the changes are on commit d8dfc78